data/systemd-user/snapd.aa-prompt-ui.service.in

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=snapd UI prompt
+
+[Service]
+Type=dbus
+BusName=io.snapcraft.Prompt
```

Comment:
should the bus name be this or AAPrompt ?

Reply:
Not really, it's only UI code, not AA specific, it could be used with other LSM as well.